ClassicImplementation/src/modules/SessionReplay/views/SessionReplayLinking.tsx

```diff
@@ -0,0 +1,45 @@
+
+import Contentsquare from '@contentsquare/react-native-bridge';
+import Clipboard from '@react-native-community/clipboard';
+import React, { useState } from 'react';
+import { Button, StyleSheet, Text, View } from 'react-native';
+import { colors, gridUnit } from '../../../constants';
+
+export const SessionReplayLinking = () => {
+  const [SRLinkDescription, setSRLinkDescription] = useState('Click on the button to retrieve the current Session Replay link');
+
+  const getCurrentSessionReplayLink = async () => {
+
+    const link = await Contentsquare.getCurrentSessionReplayLink()
+    console.log(link)
```

Comment:
Remove console.log

Reply:
Good point, I forgot to remove it.

Reply:
esLint would help here as well 👀

Reply:
Should we add esLint as a part of this PR?